core/src/main/scala/org/apache/spark/internal/io/FileCommitProtocol.scala

```diff
@@ -106,6 +106,13 @@ abstract class FileCommitProtocol {
    */
   def commitTask(taskContext: TaskAttemptContext): TaskCommitMessage
 
+  /**
+   * Commits a task which blongs to a specific stage after the writes succeed.
+   * Must be called on the executors when running tasks.
+   */
+  private[spark]
+  def commitTask(taskContext: TaskAttemptContext, stageId: Int): TaskCommitMessage
```

Comment:
Modifiers should be in the same line as method declarations, all over your patch. Just look at existing code and follow the style.